app/src/components/submissionPage/OnSubmitMessageDisplay.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render as librender } from "@testing-library/react";
+import { screen } from "@testing-library/react";
+import OnSubmitMessageDisplay from "./OnSubmitMessageDisplay";
+
+describe("OnSubmitMessageDisplay", () => {
+    test("Submit message appears if confirmation is true", () => {
```

Comment:
Using the library `jest-each` may be helpful. See [here](https://medium.com/@suman.maity112/parameterized-tests-with-jest-2932f42b1509).

Reply:
nice, will likely go through tests & combine at a later date, this particular one is now combined